components-api/src/main/scala/pl/touk/nussknacker/engine/api/properties/ScenarioProperties.scala

```diff
@@ -0,0 +1,25 @@
+package pl.touk.nussknacker.engine.api.properties
+
+import io.circe.generic.JsonCodec
+import pl.touk.nussknacker.engine.api.component.SingleScenarioPropertyConfig
+
+@JsonCodec case class ScenarioProperties(
```

Comment:
We don't need to keep it in the `components-api` module, let's move it to designer module

Reply:
Same problem with moving as the previous comment

Reply:
I can't see a connection. `designer` module depends on every -api module it shouldn't be problematic

Reply:
Classes `DeploymentManagerProvider` `DeploymentManagerStub` need this ScenarioProperties class and it's not acessible for them when moved to  `designer/server/src/main/scala/pl/touk/nussknacker/ui/definition/scenarioproperty/ScenarioProperties.scala`. Should i move them too?